lib/cucumber/glue/step_definition.rb

```diff
@@ -122,9 +122,8 @@ def ==(other)
 
       # @api private
       def arguments_from(step_name)
-        args = @expression.match(step_name)
+        @expression.match(step_name)
         # @registry.invoked_step_definition(regexp_source, location) if args
```

Comment:
Can we delete this line?

Reply:
The one that is commented?
If so, yes, I guess so :)